server/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "preserveWatchOutput": true,
     "baseUrl": "./",
     "jsx": "react",
-    "types": ["vitest/globals"]
+    "types": ["vitest/globals"],
+    "noErrorTruncation": true
```

Comment:
What is this for?

Reply:
The Kysely types can be pretty complex, so this tells VS Code to show more of the type when you hover over something.